Source/FieldSolver/FiniteDifferenceSolver/HybridPICSolveE.cpp

```diff
@@ -679,10 +676,14 @@ void FiniteDifferenceSolver::HybridPICSolveECylindrical (
                 // Add resistivity only if E field value is used to update B
                 if (solve_for_Faraday) { Ez(i, j, 0) += eta(rho_val, jtot_val) * Jz(i, j, 0); }
 
-                if (include_hyper_resistivity_term) {
+                if (include_hyper_resistivity_term && solve_for_Faraday) {
```

Comment:
This is not needed since we set:
```
const bool include_hyper_resistivity_term = (eta_h > 0.0) && solve_for_Faraday;
```
```suggestion
                if (include_hyper_resistivity_term) {
```